bedrock/firefox/templates/firefox/developer/whatsnew-fx68.html

```diff
@@ -2,32 +2,82 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% add_lang_files "firefox/whatsnew_66" %}
+{% add_lang_files "firefox/whatsnew_67" %}
```

Comment:
Not entirely sure if this is something to worry about, but if we're switching templates here does that have l10n implications?

I'm guessing locales that have `whatsnew_66.lang` and `whatsnew_67.lang` set to active may not necessarily map 1:1, and afaik release engineering only trigger each whatsnew release using a list of locales that have each page translated. I'm not sure on the numbers this may equatesto for beta and dev edition, but is there a risk some people could see a page in the wrong language here?

Reply:
Could falling back to the (more widely translated) default WNP be a viable option here, if this is an issue?

Reply:
I'm actually not sure what the best path here is... there was some kind of special rule for Dev Edition for 68 and then for 68-trailhead, and that seems to no longer be very relevant... so I'm thinking we can possibly decom both of those as and have just the single standard WNP for dev edition.

Reply:
This is a good point - those pages we're meant as a one-off thing for trailhead, and we agreed that the regular Dev Edition pages shold return after. If it simplifies this PR, I'd be in favor of retiring those one-off pages and letting the original defaults take precedence?

Reply:
I shall make it so.